resources/assets/components/Campaign/index.js

```diff
@@ -109,6 +110,12 @@ const Campaign = ({ id }) => {
           <p>–</p>
         )}
 
+        <h4>Campaign Group Type ID</h4>
+        {campaign.groupTypeId ? (
+          <a href={`/group-types/${groupTypeId}`}>{campaign.groupTypeId}</a>
```

Comment:
I missed this in the last review - I think we'd need to make this change to send to a valid URL?
```suggestion
          <a href={`/group-types/${campaign.groupTypeId}`}>{campaign.groupTypeId}</a>
```

Reply:
Surprised I missed this, got it